TSPL.docc/LanguageGuide/ControlFlow.md

```diff
@@ -1440,6 +1440,78 @@ and `distance` is an integer in both patterns ---
 which means that the code in the body of the `case`
 can always access a value for `distance`.
 
+## Patterns
```

Comment:
Reviewers: Open to alternate names for this section.  This is a bit of a placeholder heading.

Reply:
@colleenish Any suggestions or opinions on "Patterns" as a subheading?